Index: src/main/java/net/andrewcpu/elevenlabs/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package net.andrewcpu.elevenlabs;\r\n\r\nimport net.andrewcpu.elevenlabs.elements.User;\r\nimport net.andrewcpu.elevenlabs.elements.Voice;\r\nimport net.andrewcpu.elevenlabs.elements.VoiceSettings;\r\n\r\nimport java.io.File;\r\nimport java.util.List;\r\n\r\npublic class Main {\r\n\t//api key\r\n\tpublic static void main(String[] args) throws Exception{\r\n\t\tElevenLabsAPI.getInstance().setAPIKey(\"092a02471f488954fd5dd1cbf2cc355e\");\r\n\t\tList<Voice> voices = Voice.getVoices();\r\n\t\tfor(Voice voice : voices){\r\n\t\t\tSystem.out.println(voice.getVoiceId() + \", \" + voice.getName());\r\n//\t\t\tif(voice.getName().startsWith(\"Roga\")){\r\n//\t\t\t\tvoice.generate(\"Hey. This is joe rogan, coming at you live.\", new VoiceSettings(0.5, 0.5), new File(\"rogan.mp3\"));\r\n//\t\t\t}\r\n\t\t}\r\n\t\tSystem.out.println(User.getUser().toString());\r\n\r\n\t}\r\n}\r\n
===================================================================
diff --git a/src/main/java/net/andrewcpu/elevenlabs/Main.java b/src/main/java/net/andrewcpu/elevenlabs/Main.java
--- a/src/main/java/net/andrewcpu/elevenlabs/Main.java	
+++ b/src/main/java/net/andrewcpu/elevenlabs/Main.java	
@@ -1,24 +1,17 @@
 package net.andrewcpu.elevenlabs;
 
-import net.andrewcpu.elevenlabs.elements.User;
-import net.andrewcpu.elevenlabs.elements.Voice;
-import net.andrewcpu.elevenlabs.elements.VoiceSettings;
-
 import java.io.File;
+import java.util.HashMap;
 import java.util.List;
 
 public class Main {
 	//api key
 	public static void main(String[] args) throws Exception{
 		ElevenLabsAPI.getInstance().setAPIKey("092a02471f488954fd5dd1cbf2cc355e");
-		List<Voice> voices = Voice.getVoices();
-		for(Voice voice : voices){
-			System.out.println(voice.getVoiceId() + ", " + voice.getName());
-//			if(voice.getName().startsWith("Roga")){
-//				voice.generate("Hey. This is joe rogan, coming at you live.", new VoiceSettings(0.5, 0.5), new File("rogan.mp3"));
-//			}
-		}
-		System.out.println(User.getUser().toString());
-
+		HashMap<String, String> labels = new HashMap<>();
+		labels.put("accent", "American");
+		System.out.println(
+				ElevenLabsAPI.getInstance().createVoice("Andrew Stein", labels, List.of(new File("C:\\Users\\stein\\OneDrive\\Documents\\andrew\\audio_1.mp3"), new File("C:\\Users\\stein\\OneDrive\\Documents\\andrew\\audio2.mp3")))
+		);
 	}
 }
